source/refined-github.css

```diff
@@ -35,17 +35,6 @@
 	display: none !important;
 }
 
-/* Hide `New pull request button` near file list */
-.file-navigation .new-pull-request-btn {
-	display: none;
-}
-
-@keyframes fade-in {
-	from {
-		opacity: 0%;
-	}
-}
```

Comment:
Added in #735 but no longer used (I didn't bother to find when it was removed)

And we are actually overriding the native one:

```css
/* https://github.githubassets.com/assets/node_modules/@primer/css/utilities/animations.scss */
@keyframes fade-in {
  0% {
    opacity: 0;
  }

  100% {
    opacity: 1;
  }
}
```